File: WebAPI/Froom.Data/Models/RoomModel.cs
using System.Collections.Generic;
using Froom.Data.Entities;

namespace Froom.Data.Models
{
    public class RoomModel
    {
        public int Number { get; set; }
        public int Floor { get; set; }
        public string BuildingName { get; set; }
        public int Capacity { get; set; }
        public ICollection<Point> Points { get; set; }
    }
}

File: WebAPI/WebAPI/vue/types/room/Models/RoomModel.ts
import { IPoint } from '../IPoint'
import { IRoomModel } from './IRoomModel'

export class RoomModel implements IRoomModel {
  number: string = '';
  buildingName: string = '';
  campusName: string = '';
  floorNumber: string = '';
  floorOrder: number = 0;
  points: IPoint[] = [];
  capacity?: number;
}
